export-image/04-finalise/01-run.sh

```diff
@@ -95,11 +95,23 @@ else
 	make_bootable_image "${STAGE_WORK_DIR}/${IMG_FILENAME}${IMG_SUFFIX}.qcow2" "$IMG_FILE"
 fi
 
-if [ "${DEPLOY_ZIP}" == "1" ]; then
+case "${DEPLOY_COMPRESSION}" in
+zip)
 	pushd "${STAGE_WORK_DIR}" > /dev/null
-	zip "${DEPLOY_DIR}/${ZIP_FILENAME}${IMG_SUFFIX}.zip" \
+	zip "${DEPLOY_DIR}/${ARCHIVE_FILENAME}${IMG_SUFFIX}.zip" \
```

Comment:
If  we're using compression level 9 for gz and xz, it would probably make sense for zip to do so as well. I suspect people will want this to be configurable though.

Reply:
Makes sense, I'll create an option for this.

Reply:
Should the default be set to level 9 by the way ?

Reply:
Not sure that I have a strong opinion on this. One one hand, it might take quite a while without much difference in size, on the other hand, it's the sort of thing that runs as a nightly cronjob, so it doesn't matter how long it takes.

I think I'd prefer that the tools used whatever their defaults are.

Reply:
OK, they all default to  6, I'll change this then.